docs/tutorial/part-four/index.md

```diff
@@ -286,16 +265,16 @@ The basic GraphQL query that retrieves the `title` in your `about.js` changes ab
 }
 ```
 
-> 💡 In [part five](/tutorial/part-five/#introducing-graphiql), you'll meet a tool that lets us interactively explore the data available through GraphQL, and help formulate queries like the one above.
+> 💡 W [części piątej](/tutorial/part-five/#introducing-graphiql), Napotkasz się na narzędzie, które pozwoli Ci interaktywnie przeszukiwać dane dostępne poprzez GraphQL i pomoże Ci formułować zapytania, takie jak te wyżej.
 
-Page queries live outside of the component definition -- by convention at the end of a page component file -- and are only available on page components.
+Zapytania stron (page queries) znajdują się poza definicją komponentu — przyjęliśmy konwencję, aby umieszczać je na końcu pliku danego komponentu — możliwe do użycia, są tylko dla komponentów stron.
 
-### Use a StaticQuery
+### Użyj StaticQuery
 
-[StaticQuery](/docs/static-query/) is a new API introduced in Gatsby v2 that allows non-page components (like your `layout.js` component), to retrieve data via GraphQL queries.
-Let's use its newly introduced hook version — [`useStaticQuery`](/docs/use-static-query/).
+[StaticQuery (zapytanie statyczne)](/docs/static-query/) to nowe API wprowadzone w Gatsby v2 które pozwala komponentom innym niż strony (takim jak Twój komponent `layout.js`), na otrzymywanie danych poprzez zapytania GraphQL.
+Użyjmy nowo prowadzonej wersji hookowej — [`useStaticQuery`](/docs/use-static-query/).
```

Comment:
```suggestion
Użyjmy nowo wprowadzonej wersji hook — [`useStaticQuery`](/docs/use-static-query/).
```

Reply:
użyłbym albo 'wersji hook' albo 'wersji korzystającej z hooks', średnio dla mnie wygląda 'hookowej', opinie?

Reply:
widziałem, że w [polskiej dokumentacji Reacta ](https://github.com/reactjs/pl.reactjs.org/issues/3#issuecomment-462187447) odmieniamy `hook`, także tutaj też to zrobiłem.

można też napisać to tak:
```suggestion
Użyjmy nowo wprowadzonej wersji korzystającej z hooków — [`useStaticQuery`](/docs/use-static-query/).
```